### DISCUSSION TITLE
Is Winlogbeat not in 2.4 ?

### DISCUSSION THREAD

**Discussion by seckeymaker** [Security-Onion-Solutions/securityonion#12566] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

20

### Storage for /

1tb

### Storage for /nsm

1tb

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Winlogbeat is not in downloads section ?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
2.4 uses the Elastic Agent https://docs.securityonion.net/en/2.4/elastic-agent.html. There is a agent policy to bring in Windows event logs https://docs.securityonion.net/en/2.4/elastic-fleet.html#agent-policies-endpoints-initial

**Comment by seckeymaker** [comment]
Thank you.
